features/org.wso2.carbon.event.handler.notification.server.feature/resources/p2.inf

```diff
@@ -5,4 +5,5 @@ org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../r
 org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../repository/deployment/server/eventstreams); \
 org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../repository/deployment/server/eventpublishers); \
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.event.handler.notification.server_${feature.version}/id_gov_notify_stream_1.0.0.json,target:${installFolder}/../../deployment/server/eventstreams/id_gov_notify_stream_1.0.0.json,overwrite:true);\
+org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.event.handler.notification.server_${feature.version}/id_gov_sms_notify_stream_1.0.0.json,target:${installFolder}/../../deployment/server/eventstreams/id_gov_sms_notify_stream_1.0.0.json,overwrite:true);\
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.event.handler.notification.server_${feature.version}/EmailPublisher.xml,target:${installFolder}/../../deployment/server/eventpublishers/EmailPublisher.xml,overwrite:true);\
```

Comment:
add a new line at the end of the file.

Reply:
Fixed in https://github.com/wso2-extensions/identity-event-handler-notification/pull/112